tests/service/servables/srl_test.py

```diff
@@ -16,4 +16,10 @@ def test_uses_named_inputs(self):
         result = model.predict_json(inputs)
 
         # TODO(joelgrus): update this when you figure out the result format
-        assert result
+        verbs = result.get("verbs")
+
+        assert verbs  # is not None
```

Comment:
Pylint doesn't like `assert verbs != None`, and tells you to use `assert verbs is not None` instead, because of this: https://stackoverflow.com/questions/3257919/what-is-the-difference-between-is-none-and-none.